### PR TITLE
K8SPXC-947: add `backupBackoffLimit` to `cr`

### PR DESCRIPTION
**Pull request by pooknull**
[K8SPXC-947](https://jira.percona.com/browse/K8SPXC-947) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

https://jira.percona.com/browse/K8SPXC-947

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| We run 0 out of 35|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1137/commits/99b24bcce5ef2174dcc73d4160c1e52e3ec8c109
image: `perconalab/percona-xtradb-cluster-operator:PR-1137-99b24bcc`

**Review by hors — CHANGES_REQUESTED** [review]
@pooknull please check tests

**Comment by hors** [comment]
@pooknull  also please update helm chart too, we need to have possibility to set this option for helm chart.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/scheduled-backup.log) |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/upgrade-haproxy.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/haproxy.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/scaling.log) |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/big-data.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/scaling-proxysql.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/init-deploy.log) |
| cross-site | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/cross-site.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/storage.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/recreate.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/security-context.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/limits.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/restore-to-encrypted-cluster.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/self-healing.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/monitoring-2-0.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/self-healing-chaos.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/demand-backup.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/affinity.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/self-healing-advanced.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/pitr.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/one-pod.log) |
| self-healing-advanced-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/self-healing-advanced-chaos.log) |
| demand-backup-encrypted-with-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/demand-backup-encrypted-with-tls.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/auto-tuning.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/operator-self-healing.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/proxysql-sidecar-res-limits.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/operator-self-healing-chaos.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/users.log) |
| tls-issue-self | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/tls-issue-self.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/tls-issue-cert-manager.log) |
| tls-issue-cert-manager-ref | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/tls-issue-cert-manager-ref.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/upgrade-proxysql.log) |
| validation-hook | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/validation-hook.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/smart-update.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/99b24bcc/upgrade-consistency.log) |
| We run 35 out of 35|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1137/commits/99b24bcce5ef2174dcc73d4160c1e52e3ec8c109
image: `perconalab/percona-xtradb-cluster-operator:PR-1137-99b24bcc`

File: pkg/pxc/backup/job.go [hunk 1]
```diff
@@ -39,17 +39,20 @@ func (*Backup) Job(cr *api.PerconaXtraDBClusterBackup, cluster *api.PerconaXtraD
...
+	resources, err := app.CreateResources(cluster.Spec.Backup.Storages[spec.StorageName].Resources)
 	if err != nil {
 		return batchv1.JobSpec{}, fmt.Errorf("cannot parse Backup resources: %w", err)
 	}
 
 	manualSelector := true
 	backbackoffLimit := int32(10)
+	if cluster.CompareVersionWith("1.11.0") >= 0 && cluster.Spec.Backup.BackoffLimit != nil {
+		backbackoffLimit = *cluster.Spec.Backup.BackoffLimit
+	}
```
Comment (egegunes) [inline pkg/pxc/backup/job.go:52]: I believe it's time to fix this variable's name. Why not `backoffLimit`?

File: deploy/cr.yaml [hunk 2]
```diff
@@ -455,6 +455,7 @@ spec:
         cpu: 300m
   backup:
     image: perconalab/percona-xtradb-cluster-operator:main-pxc8.0-backup
+#    backupBackoffLimit: 6
```
Comment (egegunes) [inline deploy/cr.yaml:458]: Can't we just name it `backoffLimit`? @hors @spron-in

Comment (hors) [reply deploy/cr.yaml:458]: @egegunes up to @spron-in (in spec we have `backupBackoffLimit`)

Comment (spron-in) [reply deploy/cr.yaml:458]: I'm ok with backoffLimit. Please suggest the change in the spec though.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/haproxy.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/scaling.log) |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/upgrade-haproxy.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/scheduled-backup.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/storage.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/recreate.log) |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/big-data.log) |
| cross-site | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/cross-site.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/scaling-proxysql.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/self-healing.log) |
| self-healing-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/self-healing-chaos.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/init-deploy.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/restore-to-encrypted-cluster.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/security-context.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/upgrade-proxysql.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/limits.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/self-healing-advanced.log) |
| self-healing-advanced-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/self-healing-advanced-chaos.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/demand-backup.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/monitoring-2-0.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/operator-self-healing.log) |
| operator-self-healing-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/operator-self-healing-chaos.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/smart-update.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/affinity.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/one-pod.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/auto-tuning.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/pitr.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/proxysql-sidecar-res-limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/users.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/upgrade-consistency.log) |
| demand-backup-encrypted-with-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/demand-backup-encrypted-with-tls.log) |
| tls-issue-self | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/tls-issue-self.log) |
| We run 32 out of 35|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1137/commits/276035d235a139a7cf8eb4f02c055e5eee690bf2
image: `perconalab/percona-xtradb-cluster-operator:PR-1137-276035d2`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/scaling.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/scaling-proxysql.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/security-context.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/haproxy.log) |
| cross-site | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/cross-site.log) |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/upgrade-haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/init-deploy.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/recreate.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/storage.log) |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/big-data.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/scheduled-backup.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/limits.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/restore-to-encrypted-cluster.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/self-healing.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/monitoring-2-0.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/demand-backup.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/self-healing-chaos.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/affinity.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/pitr.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/one-pod.log) |
| demand-backup-encrypted-with-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/demand-backup-encrypted-with-tls.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/auto-tuning.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/proxysql-sidecar-res-limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/users.log) |
| tls-issue-self | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/tls-issue-self.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/upgrade-proxysql.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/tls-issue-cert-manager.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/smart-update.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/upgrade-consistency.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/self-healing-advanced.log) |
| self-healing-advanced-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/self-healing-advanced-chaos.log) |
| tls-issue-cert-manager-ref | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/tls-issue-cert-manager-ref.log) |
| validation-hook | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/validation-hook.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/276035d2/operator-self-healing-chaos.log) |
| We run 35 out of 35|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1137/commits/276035d235a139a7cf8eb4f02c055e5eee690bf2
image: `perconalab/percona-xtradb-cluster-operator:PR-1137-276035d2`

**Review by hors — DISMISSED** [review]

File: deploy/cr.yaml (hunk 2 above)
Comment (egegunes) [reply deploy/cr.yaml:458]: @spron-in couldn't find the spec for it, could you share the link?

@pooknull please also fix the example in CR

**Review by egegunes — CHANGES_REQUESTED** [review]
Please fix the field name in cr.yaml too

File: deploy/cr.yaml (hunk 2 above)
Comment (pooknull) [reply deploy/cr.yaml]: fixed 61c49c6b8587b763d0583bcb550b2b15981598b1

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/storage.log) |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/big-data.log) |
| cross-site | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/cross-site.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/haproxy.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/recreate.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/scheduled-backup.log) |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/upgrade-haproxy.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/self-healing.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/self-healing-chaos.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/scaling-proxysql.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/init-deploy.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/restore-to-encrypted-cluster.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/security-context.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/self-healing-advanced.log) |
| self-healing-advanced-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/self-healing-advanced-chaos.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/upgrade-proxysql.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/limits.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/demand-backup.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/monitoring-2-0.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/operator-self-healing-chaos.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/smart-update.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/affinity.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/one-pod.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/pitr.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/auto-tuning.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/proxysql-sidecar-res-limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/users.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/upgrade-consistency.log) |
| demand-backup-encrypted-with-tls | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/demand-backup-encrypted-with-tls.log) |
| tls-issue-self | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/tls-issue-self.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/tls-issue-cert-manager.log) |
| tls-issue-cert-manager-ref | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/tls-issue-cert-manager-ref.log) |
| validation-hook | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1137/61c49c6b/validation-hook.log) |
| We run 35 out of 35|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1137/commits/61c49c6b8587b763d0583bcb550b2b15981598b1
image: `perconalab/percona-xtradb-cluster-operator:PR-1137-61c49c6b`

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**